core/app.py

```diff
@@ -259,7 +259,7 @@ def _do_delete_dupe(self, dupe, link_deleted, use_hardlinks, direct_deletion):
 
     def _create_file(self, path):
         # We add fs.Folder to fileclasses in case the file we're loading contains folder paths.
-        return fs.get_file(path, self.fileclasses + [fs.Folder])
+        return fs.get_file(path, self.fileclasses + [se.fs.Folder] + [fs.Folder])
```

Comment:
Replacing `fs.Folder` with `se.fs.Folder` should be fine in all cases from what I can tell.

Reply:
I suppose you're right. It seems like will return the first `fileclass.can_handle(path)` anyway...